Header header
# Array of observations
float32 robot_theta
float32 robot_phi
float32 robot_psi
float32 robot_theta_dot
float32 robot_phi_dot